### ISSUE TITLE
Fresh install does not succesfully open

### ISSUE THREAD

**Issue by Azgrom** [spyder-ide/spyder#15422]
# Problem Description

Hi there!

I use Manjaro KDE and whenever I install and try to run spyder I get the following error:
```bash
$ spyder
No QCoreApplication instance found. Application patches not applied. You have to call load_stylesheet function after instantiation of QApplication to take effect. 
Traceback (most recent call last):
  File "/home/rafael/anaconda3/envs/Sch-JLab/bin/spyder", line 11, in <module>
    sys.exit(main())
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/app/start.py", line 214, in main
    mainwindow.main(options, args)
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 2155, in create_window
    main.pre_visible_setup()
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1186, in pre_visible_setup
    plugin_instance.before_mainwindow_visible()
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/plugins/layout/plugin.py", line 133, in before_mainwindow_visible
    self.setup_layout(default=False)
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/plugins/layout/plugin.py", line 237, in setup_layout
    self.set_window_settings(*settings)
  File "/home/rafael/anaconda3/envs/Sch-JLab/lib/python3.9/site-packages/spyder/plugins/layout/plugin.py", line 417, in set_window_settings
    self.window_position = QPoint(pos[0], pos[1])  # x,y
TypeError: arguments did not match any overloaded call:
  QPoint(): too many arguments
  QPoint(int, int): argument 1 has unexpected type 'str'
  QPoint(QPoint): argument 1 has unexpected type 'str'
```
## What steps will reproduce the problem?
I usually do the installation at a conda environment. If I try to install and run spyder through pacman I get the same result.

It loads that little green loading window and after it has apparently loaded everything it disappears leaving the above terminal the above log at the terminal.

What kind of error is this?

### Dependencies

Whatever packages are downloaded and installed via `conda install spyder` or `pacman -S spyder`, for the last 4 weeks

**Comment by ccordoba12** [comment]
Hey @Azgrom, thanks for reporting. This problem should be fixed in our 5.0.1 version, released a week and a half ago and soon to be part of Anaconda.

In the meantime, please downgrade to our 4.2.5 version by running

    conda install spyder=4.2.5 qdarkstyle=2.8.1 spyder-kernels=1.10.2

**Comment by Azgrom** [comment]
Thank you @ccordoba12!
